arangosh/Restore/RestoreFeature.cpp

```diff
@@ -898,24 +898,28 @@ arangodb::Result processInputDirectory(
         }
       }
     }
-    std::sort(collections.begin(), collections.end(), ::sortCollections);
 
-    std::vector<std::unique_ptr<arangodb::RestoreFeature::JobData>> jobs(
-        collections.size());
+    // order collections so that prototypes for distributeShardsLike come first
+    std::sort(collections.begin(), collections.end(), ::sortCollectionsForCreation);
```

Comment:
why don't you just sort "_users" so its last in the list ?

Reply:
because we distribute shards like _users in atomic db creation => it has to be created first, but filled last.

Reply:
Plus `_users` needs to be populated _after_ all other operations have finished.
It needs to be the very last operation carried out by arangorestore, because otherwise the connection credentials may be wrong afterwards